build.sbt

```diff
@@ -646,16 +648,26 @@ lazy val http4s = projectMatrix
         moduleName.value + "-ce2"
       else moduleName.value
     },
-    Test / allowedNamespaces := Seq(
-      "smithy4s.hello",
-      "alloy.test"
+    Test / allowedNamespaces := Seq("smithy4s.hello"),
+    Test / complianceTestDependencies := Seq(
+      Dependencies.Alloy.`protocol-tests`
     ),
     Test / smithySpecs := Seq(
       (ThisBuild / baseDirectory).value / "sampleSpecs" / "hello.smithy"
     ),
     Test / smithy4sSkip := Seq("openapi"),
-    Test / smithy4sDependencies := Seq(Dependencies.Alloy.`protocol-tests`),
-    (Test / sourceGenerators) := Seq(genSmithyScala(Test).taskValue)
+    (Test / sourceGenerators) := Seq(genSmithyScala(Test).taskValue),
+    (Test / resourceGenerators) := Seq(dumpModel(Test).taskValue),
+    (Test / envVars) ++= {
+      val files: Seq[File] =
+        (Test / resourceGenerators) {
+          _.join.map(_.flatten)
+        }.value
+
+      files.foldLeft(Map.empty[String, String]) { case (acc, file) =>
```

Comment:
basically, if I understand correctly, this means that if `files` contains multiple files, only the last value will be in `MODEL_DUMP`

if that's correct, maybe a `files.lastOption.map(...).getOrElse(Map.empty)` will me more explicit?

Reply:
good point